SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
+{
+	write-host "installing packages"
+	add-appxpackage "$packagePath\Microsoft.WindowsAppRuntime.Framework\Microsoft.WindowsAppRuntime.Framework.msix"
```

Comment:
I think this can be useful for all devs working in any WinAppSDK feature, not only Notifications.

If anyone works in PowerNotifications, why would they want to install the LRP?

Is it LRP the only other extra package that is installed to test any of the other WinAppSDK features?

Reply:
Adding options to install the demo packages would be wonderful!